src/locale/lang/vi.js

```diff
@@ -87,18 +87,18 @@ export default {
       confirmFilter: 'Xác nhận',
       resetFilter: 'Làm mới',
       clearFilter: 'Xóa hết',
-      sumText: 'Sum' // to be translated
+      sumText: 'Tổng' // to be translated
     },
     tree: {
       emptyText: 'Không có dữ liệu'
     },
     transfer: {
       noMatch: 'Dữ liệu không phù hợp',
       noData: 'Không tìm thấy dữ liệu',
-      titles: ['List 1', 'List 2'], // to be translated
-      filterPlaceholder: 'Enter keyword', // to be translated
-      noCheckedFormat: '{total} items', // to be translated
-      hasCheckedFormat: '{checked}/{total} checked' // to be translated
+      titles: ['Danh sách 1', 'Danh sách 2'], // to be translated
```

Comment:
Thank you all for your contribution. Can you remove the `to be translated` comments? So that other guys know that it has been translated.